package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -165,3 +176,212 @@ def _read_first_frame(self):
         self.ts.frame = 0
         if self.convert_units:
             self.convert_pos_from_native(self.ts._pos)  # in-place !
+            try:
+                self.convert_velocities_from_native(self.ts._velocities)  # in-place !
+            except AttributeError:
+                pass
+
+class DATAWriter(base.Writer):
+    """Write out the current time step as a LAMMPS DATA file.
+
+    This writer supports the sections Atoms, Masses, Velocities, Bonds,
+    Angles, Dihedrals, and Impropers. This writer will write the header
+    and these sections (if applicable). Atoms section is written in the
+    "full" sub-style if charges are available or "molecular" sub-style
+    if they are not. Molecule id is set to 0 for all atoms.
+
+    .. Note::
+
+       This writer assumes "conventional" or "real" LAMMPS units where length
+       is measured in Angstroms and velocity is measured in Angstroms per
+       femtosecond. To write in different units, specify `lengthunit`
+    """
+    format = 'DATA'
+
+    def __init__(self, filename, convert_units=None, **kwargs):
+        """Set up a DATAWriter
+
+        :Arguments:
+           *filename*
+              output filename
+        """
+        self.filename = util.filename(filename, ext='data')
+
+        if convert_units is None:
+            convert_units = flags['convert_lengths']
+        self.convert_units = convert_units
+
+        self.units = {'time': 'fs', 'length': 'Angstrom'}
+        self.units['length'] = kwargs.pop('lengthunit', self.units['length'])
+        self.units['time'] = kwargs.pop('timeunit', self.units['time'])
+        self.units['velocity'] = kwargs.pop('velocityunit', \
```

Comment:
You don't need the backslash for line continuations, just make the bracket line up [like here](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)